docs/governance/community-guidelines/releases/cmb.md

```diff
@@ -0,0 +1,68 @@
+# Change Management Board
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines and defines the guidelines for the Change Management Board (CMB) for the Common Cloud Controls (CCC) project.
+
+The Change Management Board (CMB) is a collaborative body composed of representatives from multiple banks of varying sizes and other external organizations. Its primary role is to oversee and approve changes and new catalogs that are within the Release Candidate. The CMB acts as a governing entity, facilitating discussion and arbitration when necessary, to ensure that all stakeholder concerns are addressed before approving release candidates for official publication. This diverse representation ensures that the framework is adaptable to the needs of a wide range of institutions while maintaining consistency and integrity across the board.
+
+## Responsibilities
+
+The key responsibilities of the CMB include:
+
+1. **Review of Proposed Changes**
+
+   - The CMB reviews all proposed changes to the CCC framework, ensuring they meet security and compliance standards, and align with both internal and external regulations.
+
+2. **Approval of Release Candidates**
+
+   - The board evaluates release candidates and either approves them for official release or requests further changes.
+
+3. **Arbitration of Change Requests**
+
+   - The CMB arbitrates any disputes or unresolved issues related to proposed changes, ensuring that all stakeholders reach a consensus before final approval.
+
+4. **Monitoring and Adapting to Industry Standards**
+   - The CMB ensures that the CCC framework remains aligned with evolving industry standards, adjusting guidelines as needed to stay compliant with new regulations and technological advancements.
+
+## Membership
+
+The CMB is comprised of representatives from the following:
+
+1. **FSIs of Varying Sizes**
+
+   - Representatives from banks of different scales, ensuring that the interests and needs of both small and large financial institutions are accounted for.
+
+2. **External Organizations**
+   - Representatives from various other organizations also welcomed to join this effort.
+
```

Comment:
```suggestion
### Qualifications for Participation

Individuals of any background or experience level may participate in a review.

To approve or request changes, an individual must be an appointed CMB member in good standing.

CMB members are appointed by the [Delivery WG], through application or recruitment. If you have questions, please reach out to a current WG member or join a community call.

### Release Manager Qualifications

A release manager shall be a [Delivery WG] approver or a CMB member who has provided feedback on a previous release cycle.

Release managers are expected to demonstrate the following qualities:

- Strong written communication skills
- High attention to detail
- Commitment to process and protocol
- Ability to parse and relay complex feedback
- Fundamental knowledge of the domain featured in the release
- Availability and responsiveness during the release cycle (at least one month)

### Breach of Decorum

Members of the Change Management Board are expected to follow the [FINOS Community Code of Conduct](https://community.finos.org/docs/governance/code-of-conduct) at all times.

Appointments shall be permanently revoked in the following cases:

- Repeat disrespectful communication
- Repeat obstructive behavior such as vague or non-actionable feedback
- Repeat abandonment of a stated commitment
- Undermining the process, such as deliberately circumventing or disregarding documented norms
```

Reply:
@eddie-knight we'll need to define an application process for those who are interested. maybe have an interview or something included.